components/collection/activity/ownerInsightsTabs/format.ts

```diff
@@ -0,0 +1,4 @@
+export default function (number: number) {
+  const isInteger = Number.isInteger(number)
+  return isInteger ? number.toString() : number.toFixed(4)
+}
```

Comment:
do we need to create a new file for this function?

Reply:
move it to utils